var/spack/repos/builtin/packages/ispc/package.py

```diff
@@ -0,0 +1,84 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Ispc(Package):
+    """Intel SPMD (Single Program, Multiple Data) Program Compiler.
+
+    A compiler for the C-based SPMD programming language.  It is built on
+    top of the llvm stack.
+
+    Note: this library requires a 32-bit version of `glibc` (specifically,
+    it needs to #include <gnu/stubs-32.h>).
+    """
+    homepage = "http://ispc.github.io/"
+    url      = "https://github.com/ispc/ispc/"
```

Comment:
You should use the actual tarball for the URL. Then you won't have to specify it below:
https://github.com/ispc/ispc/archive/v1.9.1.tar.gz